js/ui/autocomplete.d.ts

```diff
@@ -1,9 +1,9 @@
 import {
-    EventInfo,
-    NativeEventInfo,
-    InitializedEventInfo,
-    ChangedOptionInfo,
-    ItemInfo,
+  EventInfo,
+  NativeEventInfo,
+  InitializedEventInfo,
+  ChangedOptionInfo,
+  ItemInfo, NativeInputEventInfo,
```

Comment:
```suggestion
  ItemInfo,
  NativeInputEventInfo,
```
To keep consistency. Same for other cases